tests.php

```diff
@@ -0,0 +1,114 @@
+<?php
+	/* Tests script for phpcs-security-audit */
+
+	//eval($_GET['a']);
+	echo "aaaa";
+	echo "bbbb" . $_POST['b'];
+	echo "b";
+	db_query($_GET['a']);
+	preg_replace("/.*/ei", 'aaaaaaa', 'bbbbb');
+	preg_replace("/.*/ei", $_GET['a'], 'aaaaaaa');
+	preg_replace($_GET['b'], $_GET['a'], $_GET['c']);
+	preg_replace($b, $_GET['a'], 'aaaaaa');
+	preg_replace("aaa", $_GET['a'], 'ababaaa');
+	
+
+	// BadFunctions
+	md5();
+	phpinfo();
+	create_function($a);
+	ftp_exec($a);
+	fread($a);
+	array_map($a);
+	`$a`;
+	`$_GET`;
+	include($a);
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> error - No file extension has been found in a include/require function. This implies that some PHP code is not scanned by PHPCS.